src/main/resources/templates/admin.html

```diff
@@ -20,7 +20,7 @@ <h1 class="text-center text-md-left">UH Groupings Administration</h1>
                         Manage Admins
                     </a>
                 <li class="nav-item" role="presentation">
-                    <a href="#manage-person" id="manage-person-tab" data-toggle="tab" class="nav-link tab-color" role="tab">
+                    <a href="#manage-subject" id="manage-subject-tab" data-toggle="tab" class="nav-link tab-color" role="tab">
                         Manage Person
```

Comment:
Since the Manage Person tab is only used by a small number of people, let's go ahead and rename the text as well to Manage Subject.